draft-ietf-mls-protocol.md

```diff
@@ -982,12 +946,7 @@ In order to facilitate asynchronous addition of participants to a
 group, it is possible to pre-publish initialization keys that
 provide some public information about a user or group.  UserInitKey
```

Comment:
since we no longer have groupInitKeys, should we remove the "or group" part of the sentence above ?